tools/sggolangcilint/golangci.yml

```diff
@@ -76,9 +83,9 @@ linters:
     # [fast: true, auto-fix: false]
     - goprintffuncname
 
-    # Check for code that can be simplified.
+    # Inspects source code for security problems.
     # [fast: false, auto-fix: false]
-    - gosimple
```

Comment:
why is this removed in the new config?

Reply:
Should be moved.
![image](https://github.com/user-attachments/assets/5dc273f7-c9c1-46f1-b40b-9d0114a4f87a)